carbonmark/pages/users/[user].tsx

```diff
@@ -58,11 +57,13 @@ export const getStaticProps: GetStaticProps<PageProps, Params> = async (
 
     // Haven't fetched carbonmark API yet?
     if (!carbonmarkUser) {
-      const userData = await getCarbonmarkUser({
-        user: userAddress,
-        type: "wallet",
+      const response = await client["/users/{walletOrHandle}"].get({
```

Comment:
FYI api fails without `?type=wallet`, just slap that `query` back in there and we should be good
![image](https://github.com/KlimaDAO/klimadao/assets/88635679/8ebe97b2-0cec-4f9d-827f-c621d8894a1f)

Reply:
Ah true, done